lib/nats.js

```diff
@@ -1334,12 +1344,149 @@ Client.prototype.request = function(subject, opt_msg, opt_options, callback) {
  * @param {String} opt_msg
  * @param {Object} opt_options
  * @param {Function} callback - can be called with message or NatsError if the request timed out.
```

Comment:
Missing timeout from list of parameters here (may want to indicate that it is required)